crates/common/tedge_config/src/tedge_config_dto.rs

```diff
@@ -88,3 +91,10 @@ pub(crate) struct MqttConfigDto {
 pub(crate) struct SoftwareConfigDto {
     pub(crate) default_plugin_type: Option<String>,
 }
+
+#[derive(Debug, Default, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct TmpPathConfigDto {
+    #[serde(default)]
+    pub(crate) path: Option<FilePath>,
+}
```

Comment:
We add here a set of config related to `Tmp` with a single property, the `path`.
I wonder if doing the reverse would be better: A config set related to `Path` with a `tmp` property.
The latter is more likely to be extended with misc paths, while the former will be a singleton for ever.